homeassistant/components/unifi/sensor.py

```diff
@@ -79,6 +79,8 @@ def async_wlan_client_value_fn(controller: UniFiController, wlan: Wlan) -> int:
             client.mac
             for client in controller.api.clients.values()
             if client.essid == wlan.name
+            and dt_util.utcnow() - dt_util.utc_from_timestamp(client.last_seen or 0)
```

Comment:
This would have been cheaper to use time.time() and cache the static value.  Not sure if it's run enough to matter but not a blocker for sure